x-pack/test/functional/apps/observability_log_explorer/app.ts

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import expect from '@kbn/expect';
+import { FtrProviderContext } from '../../ftr_provider_context';
+
+export default function ({ getService, getPageObjects }: FtrProviderContext) {
+  const kibanaServer = getService('kibanaServer');
+  const PageObjects = getPageObjects(['common', 'navigationalSearch', 'observabilityLogExplorer']);
+  const testSubjects = getService('testSubjects');
+
+  describe('Application', () => {
+    before('initialize tests', async () => {
+      await kibanaServer.importExport.load('test/functional/fixtures/kbn_archiver/discover');
+    });
+
+    after('clean up archives', async () => {
+      await kibanaServer.importExport.unload('test/functional/fixtures/kbn_archiver/discover');
+    });
```

Comment:
Do we need these archives for the purpose of the test or can be removed?

Reply:
I remember they were necessary to at least have a data view created and render the discover search bar including the customizations, but for the search tests only they shouldn't be necessary.

I tried removing them locally and the tests pass correctly 👍

Reply:
I see, thanks for background info